<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mafOperationWidget</class>
 <widget class="QWidget" name="mafOperationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="boxOperation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Operation Name</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>277</width>
               <height>339</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayoutOperation"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonOkCancel">
       <property name="toolTip">
        <string>Press Ok to start the operation, Cancel to close it.</string>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>execute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>mafOperationWidget</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>operationDismissed()</signal>
  <slot>execute()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
